guides/common/modules/proc_republishing-repository-and-content-view-metadata.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Republishing_Repository_and_Content_View-Metadata_{context}"]
+= Republishing Repository and Content View Metadata
+
+In the case of republishing repository metadata, this is used when a repository distribution does not have the content that should be distributed based on the contents of the repository.
+
+This procedure should be used with caution.
+In most cases, a complete repository sync or publishing a new Content View version is recommended.
+
+.Procedure
+
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Select the product that includes the repository for which you want to republish metadata.
+Navigate to the repository where you want to republish metadata.
+. To republish metadata for the repository, select *Republish Repository Metadata* from the *Select Action* menu.
++
+[NOTE]
+====
+This action is not available for repositories that implement the *Complete Mirroring* policy since the metadata is copied from the repository's upstream source instead of being generated locally.
+====
+
+To republish Content View metadata:
+
+. In the {ProjectWebUI}, navigate to *Content View* > *Version* > *Actions*.
+. Republishing repository metadata will regenerate metadata for all repositories in the Content View version that do not adhere to the *Complete Mirroring* policy.
```

Comment:
Isn't this missing a step in between?

Reply:
@sjha4 Is there a step missing in between here?

Reply:
So you go to CV > Version > Details > Actions kebab and the action is there.
Screenshot:
![Screenshot from 2023-06-26 14-54-42](https://github.com/theforeman/foreman-documentation/assets/21146741/979dbe0b-5217-46aa-8806-1fa91f415485)

Reply:
Steps updated.